[CK_TILE] Update example README files & fix script compatibility issue (#1548)
ROCm/composable_kernel: example/ck_tile/02_layernorm2d/README.md

```
# in the root of ck_tile
mkdir build && cd build

> bash ../script/cmake-ck-dev.sh  ../ <arch>  # you can replace this <arch> to gfx90a, gfx942...

Reviewer: I'm assuming the architecture needs to be specified. If it doesn't then the wording should be different. ("you can optionally specify the architecture" or something like that)

```suggestion
bash ../script/cmake-ck-dev.sh  ../ <arch>  # replace <arch> with the appropriate architecture, for example gfx90a or gfx942
```

Author: Thank you. Yes, the arch is optional, so I agree that we need specify this in the doc
